documentation/docs/components/operators/nodes/example_accordion.tsx

```diff
@@ -0,0 +1,20 @@
+import {Accordion, AccordionItem} from "@nextui-org/react";
```

Comment:
are we using this component? if not - do we need to include it?

Reply:
It's the only collapsible component I found, but it looks ugly, not sure if it's theming or what, do you have a minute to try it?

Reply:
Will leave in for now, lets get it in the man new branch and then we can take it from there